src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -39,7 +39,7 @@
     <_TargetFrameworkDirectories>$(MSBuildThisFileDirectory)/Documentation</_TargetFrameworkDirectories>
     <_FullFrameworkReferenceAssemblyPaths>$(MSBuildThisFileDirectory)/Documentation</_FullFrameworkReferenceAssemblyPaths>
     <SkipCommonResourcesIncludes>true</SkipCommonResourcesIncludes>
-    <LangVersion>7.2</LangVersion>
+    <LangVersion>7.3</LangVersion>
```

Comment:
We may just want to do `latest` here.

Reply:
We shouldn't do this until we are done with 2.1. We don't want to depend on 7.3 yet as it is still under development.

Reply:
I do not think we will take this PR for 2.1 for other reasons.

But I do not see a fundamental problem with always using latest.

Reply:
>I do not think we will take this PR for 2.1 for other reasons.

If that is the case we shouldn't merge this until after our final merge into release/2.1 which is scheduled for this friday.